lib/SiftClient.php

```diff
@@ -2,130 +2,350 @@
+     *           returned for every abuse_type to which you are subscribed.
+     *     - int 'timeout': By default, this client's timeout is used.
+     *     - string 'version': By default, this client's version is used.
+     *     - string 'path': The URL path to use for this call.  By default, the path for the
+     *           requested version of the Events API is used.
      *
-     * @param $userId A user's id. This id should be the same as the user_id used in event calls.
-     * This parameter is required.
-     * @param $timeout (optional) The number of seconds to wait before failing the request. By default this is
-     * configured to 2 seconds (see above).
      * @return null|SiftResponse
      */
-    public function score($userId, $timeout = self::DEFAULT_TIMEOUT) {
+    public function score($userId, $opts = array()) {
+        $this->mergeArguments($opts, array(
+            'abuse_types' => array(),
+            'timeout' => $this->timeout,
+            'version' => $this->version
+        ));
+
         $this->validateArgument($userId, 'user id', 'string');
 
-        $properties = array('api_key' => $this->api_key);
+        $params = array('api_key' => $this->api_key);
+        if ($opts['abuse_types']) $params['abuse_types'] = implode(',', $opts['abuse_types']);
+
         try {
-            $request = new SiftRequest(self::userScoreApiUrl($userId), SiftRequest::GET, $properties, $timeout);
+            $request = new SiftRequest(self::userScoreApiUrl($userId, $opts['version']),
+                SiftRequest::GET, $opts['timeout'], $opts['version'], array('params' => $params));
             return $request->send();
         } catch (Exception $e) {
             return null;
         }
     }
 
+
     /**
      * Labels a user as either good or bad through the Sift Science API.
-     * Check https://siftscience.com/resources/references/labels_api.html for valid $properties fields.
      *
-     * @param $userId A user's id. This id should be the same as the user_id used in event calls.
-     * This parameter is required.
-     * @param $properties An array of name-value pairs that specify the label attributes. This parameter is required.
-     * @param $timeout (optional) The number of seconds to wait before failing the request. By default this is
-     * configured to 2 seconds (see above).
+     * See https://siftscience.com/resources/references/labels_api.html for valid $properties
+     * fields.
+     *
+     * @param string $userId  The ID of a user.  This parameter is required.
+     *
+     * @param $properties An array of name-value pairs that specify the label attributes. This
+     *     parameter is required.
+     *
+     * @param array $opts  Array of optional parameters for this request:
+     *     - int 'timeout': By default, this client's timeout is used.
+     *     - string 'version': By default, this client's version is used.
+     *
```

Comment:
Longer term, I think the 'path' parameter is weird and we should try to remove it from the `track` method rather than add it in more places.